static/js/unscrambleEmail.js

```diff
@@ -0,0 +1,17 @@
+import { unscramble } from "botex";
+
+export function onRouteDidUpdate({ location, previousLocation }) {
+  if (typeof window !== 'undefined') {
+    setTimeout(() => {
```

Comment:
Do we really need a timeout here? Docusaurus docs [say](https://docusaurus.io/docs/advanced/client#client-module-lifecycles): 

> If you need to manipulate the new page's DOM, you'll likely want to use `onRouteDidUpdate`, which will be fired as soon as the DOM on the new page **has mounted**.

Reply:
Yes , without timeout it was not working

Reply:
I experimented a bit, and the following seems to work well (provided that email links refer to `/#contact-email` instead of `#contact-email`):

```js
import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment';
import { unscramble } from "botex";

const botexKey = "CHKXRg4cdMZ0XADF";
const obfuscatedEmail = "BwC1311181fB4C1sBzmC14162k1dByC112z16132t1o1jBh";

export function onRouteUpdate({ location, previousLocation }) {
  if (ExecutionEnvironment.canUseDOM) {
    if (location.pathname === '/' && location.hash === '#contact-email') {
      const email = unscramble(obfuscatedEmail, botexKey);
      window.location.href = `mailto:${email}`;
    }
  }
}
```